practica-4.html

```diff
@@ -3,7 +3,7 @@
   <head>
     <!-- Required meta tags -->
     <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="viewport" content="width: 30px; height: 30px;-width: 30px; height: 30px;-scale=1, shrink-to-fit=no">
```

Comment:
Why the change on the width and height?

Reply:
Sorry a made a change on the width and height on some lines under that ones in a group but i didn't realize that the meta tag was change.